apps/aechannel/test/aesc_fsm_SUITE.erl

```diff
@@ -1863,13 +2052,20 @@ create_channel_from_spec(I, R, Spec, Port, Debug, Cfg) ->
     create_channel_from_spec(I, R, Spec, Port, false, Debug, Cfg).
 
 create_channel_from_spec(I, R, Spec, Port, UseAny, Debug, Cfg) ->
-    RProxy = spawn_responder(Port, Spec, R, UseAny, Debug),
-    IProxy = spawn_initiator(Port, Spec, I, Debug),
-    log("RProxy = ~p, IProxy = ~p", [RProxy, IProxy]),
-    #{ i := #{ fsm := FsmI } = I1
-     , r := #{ fsm := FsmR } = R1 } = Info
-        = match_responder_and_initiator(RProxy, Debug),
-    log(Debug, "channel paired: ~p", [Info]),
+    %% TODO: Somehow there is a CI only race condition which occurs only in 2 tests cases
+    %% I was unable to reproduce the issue locally for 2 days :(
+    %% For now just wrap this operation in a retry loop and come back to it later
+    {FsmI, FsmR, I1, R1} = retry(3, 100,
```

Comment:
The test cases should be documented here. Also could you please make the retry conditionally such that we can easily run the tests without retries to check for these failures?